middlewares/validators/userDeviceInfo.js

```diff
@@ -0,0 +1,21 @@
+const joi = require("joi");
+
+const storeUserDeviceInfo = async (req, res, next) => {
+  const schema = joi.object().strict().keys({
+    user_id: joi.string().required(),
+    device_info: joi.string().required(),
+    device_id: joi.string().required(),
```

Comment:
any specific reason for snake casing here. I have seen we are using camel Casing for Object key names at other places

Reply:
i think its a `mixed bag`. if we check the `users.js` and `recruiter.js` files inside validators they also have `snake casing`. Also i am following the API contract which was created. [here](https://github.com/Real-Dev-Squad/website-api-contracts/pull/134/files)